source/content/guides/decoupled-sites/03-starter-kits.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Pantheon Decoupled
+subtitle: 
+description: 
+categories: [platform]
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan]
+reviewed: "2022-07-31"
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled-sites/starter-kits
+anchorid: starter-kits
+---
+
+# Creating a New Project for Decoupled 
```

Comment:
Should it be more explicit between "back-end" and "front-end" projects?
maybe something like the following?
```suggestion
# Creating a New CMS Backend for a Decoupled Frontend
```